facilitator/src/idl.rs

```diff
@@ -649,6 +649,7 @@ pub struct SumPart {
     pub aggregation_start_time: i64,
     pub aggregation_end_time: i64,
     pub packet_file_digest: Vec<u8>,
+    pub total_individual_clients: i64,
```

Comment:
I believe you'll also need to update the `read` and `write` methods on `SumPart` to serialize and deserialize this field. The tests at the bottom of this file should also be updated to read/write this field.